### ISSUE TITLE
state_color not working

### ISSUE THREAD

**Issue by SupremeSports** [thomasloven/lovelace-fold-entity-row#103]
A device (switch, sensor, etc.), that has some custom icon_color, works perfectly out of the fold card, but inside the fold card, no more colors
![Untitled](https://user-images.githubusercontent.com/13887820/82731866-a518f780-9cd7-11ea-83db-80b01f9023df.jpg)

The first Pool Pump is the normal switch with colors set up. The second one is a binary sensor with colors set up. then the third one is the same switch as the top one and has colors set up

```
    - type: entities
      title: Pool pump and heater
      show_header_toggle: false
      state_color: true
      entities:
        - switch.pool_pump
        - type: custom:fold-entity-row
          head: binary_sensor.poolpump_status
          entities:
          - switch.pool_pump
          - input_select.poolpump_mode
          - type: custom:fold-entity-row
            head:
              type: section
              label: Automations
            entities:
              - automation.pool_pump_control
```

**Comment by SupremeSports** [comment]
If you look at the device, you see that color is correct, and it was working in 0.109.x

![Untitled](https://user-images.githubusercontent.com/13887820/82740421-8b4ad500-9d16-11ea-93b4-4df21704abee.jpg)

**Comment by SupremeSports** [comment]
Ok, it appears to be a much more complex issue that has NOTHING to do with fold-entity card. It has to do with HA icon managing. So, in the example I put above, it was a switch and switches were still working and that's why it was still working outside the fold-entity.Once inside the fold-entity, it's using custom-UI and this was broken. 

On the other hand, sensors, vacuum, locks, binary-sensors, etc. were not working either. But since almost all my stuff is under fold-entity, I wrongfully assumed it was a card issue. after pulling some sensors out of the card, I realized it was much more complex.

After many hours working on this, I've been able to get a fix/workaround to get it to work with Thomas' suggestion in [here](https://github.com/andrey-git/home-assistant-custom-ui/issues/184#issuecomment-634395655)
